telemetry/opting-out-of-telemetry.adoc

```diff
@@ -5,8 +5,15 @@ include::modules/common-attributes.adoc[]
 
 toc::[]
 
+ifdef::openshift-enterprise,openshift-origin[]
 You might need to opt out of Telemetry for your cluster. For example, you might need to comply with privacy laws.
+endif::[]
 
+ifdef::openshift-dedicated[]
+On OpenShift Dedicated, Telemetry is always enabled. You cannot opt out.
```

Comment:
s/OpenShift Dedicated/{product-title}/

Reply:
Isn't it "OpenShift Container Platform" but "OpenShift Dedicated"?

Reply:
Within an `openshift-dedicated`, `{product-title}` will render as "OpenShift Dedicated".

Reply:
:man_facepalming: Done. If you the result looks good, can you please merge?